core/interfaces.go

```diff
@@ -155,6 +162,12 @@ type StorageAdder interface {
 	AddPendingAuthorizations(ctx context.Context, req *sapb.AddPendingAuthorizationsRequest) (*sapb.AuthorizationIDs, error)
 	SetOrderError(ctx context.Context, order *corepb.Order) error
 	RevokeCertificate(ctx context.Context, req *sapb.RevokeCertificateRequest) error
+	// New authz2 methods
+	NewAuthorization(ctx context.Context, req *corepb.Authorization) (*sapb.AuthorizationID2, error)
```

Comment:
These were added when this change was originally only going to add the add methods. At the time I think I disliked the current naming scheme (`NewPendingAuthorization`/`AddPendingAuthorizations`) and wanted to move to something move consistent.

I'd still like to make that naming change, and since these RPCs aren't yet in use I think it's better to do that now than wait until they are and the migration process is much more painful. But if there are objections I'm not that stuck on it.

Reply:
I agree with the name change, but I think they should have a "2" at the end like their neighbors, right?

Reply:
Ah, the "2"s were mainly just to differentiate between the original methods although now I realize it may've come across as indicating they were v2 methods. Probably makes sense to include it for these then, I was hoping we'd be able to just trim the 2 once we remove all of the old methods, but I guess that'd actually end up being a bit of a pain to do :/